feedback/models.py

```diff
@@ -7,15 +7,14 @@
 
 from django.db import models, transaction
 from django.conf import settings
-from django.core.exceptions import ObjectDoesNotExist
 from django.utils import timezone
 from django.utils.functional import cached_property
 from django.utils.translation import get_language, gettext_lazy as _
 from django_colortag.models import ColorTag
 from r_django_essentials.fields import Enum
 
-from aplus_client.django.models import (
-    ApiNamespace as Site, # mooc-jutut refers api namespaces as sites
+from aplus_client.django.models import ( # noqa: F401 this is required by accounts/models.py
+    ApiNamespace as Site,
```

Comment:
FIXME NOW don't remove the comment `# mooc-jutut refers api namespaces as sites`